Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -177,6 +191,16 @@ private static string CreateSerializedHeader()
             return NewtonsoftJsonSerializer.Instance.Serialize(header);
         }
 
+        /// <summary>Converts the PKCS8 PrivateKey to RSA Parameters. Uses the Bouncy Castle library.</summary>

```

Comment:
Please change to:
Converts the PKCS8 private key to RSA parameters. This method uses the Bouncy Castle library.